tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -1800,23 +1822,169 @@ public:
    }
 
    /**
-    * @brief Create a new SnapshotHelperMT with a different output file name
+    * @brief Create a new SnapshotTTreeHelperMT with a different output file name
+    *
+    * @param newName A type-erased string with the output file name
+    * @return SnapshotTTreeHelperMT
+    *
+    * This MakeNew implementation is tied to the cloning feature of actions
+    * of the computation graph. In particular, cloning a Snapshot node usually
+    * also involves changing the name of the output file, otherwise the cloned
+    * Snapshot would overwrite the same file.
+    */
+   SnapshotTTreeHelperMT MakeNew(void *newName)
+   {
+      const std::string finalName = *reinterpret_cast<const std::string *>(newName);
+      return SnapshotTTreeHelperMT{fNSlots,           finalName,          fDirName, fTreeName,
+                                   fInputBranchNames, fOutputBranchNames, fOptions, std::vector<bool>(fIsDefine)};
+   }
+};
+
+#ifdef R__HAS_ROOT7
+void ValidateSnapshotRNTupleOutput(const RSnapshotOptions &opts, const std::string &ntupleName,
```

Comment:
The function is missing a doc string and the name seems at odd with the functionality.  Most often a `validate` routine would return a 'bool' and leave its input unchanged (eg. `IsValid`)

Reply:
Was this addressed?

Reply:
Yes, in [this commit](https://github.com/enirolf/root/commit/1d1aa6b66583bfd68509fe05c475f1e85539136b)